packages/styles/impact-badge.css

```diff
@@ -0,0 +1,39 @@
+:root {
+  --impact-badge-critical-background-color: #fea7a6;
+  --impact-badge-critical-border-color: var(--accent-error-disabled);
+
+  --impact-badge-serious-background-color: var(--accent-caution);
+  --impact-badge-serious-border-color: var(--accent-warning);
+
+  --impact-badge-moderate-background-color: var(--issue-moderate);
+  --impact-badge-moderate-border-color: #eb94ff;
+
+  --impact-badge-minor-background-color: var(--issue-minor);
+  --impact-badge-minor-border-color: var(--gray-30);
+
+  --impact-badge-text-color: var(--gray-90);
```

Comment:
This is almost there.

All of the background colors should be the appropriate `--issue-[type]` colors. This is done so that if someone uses the issue colors elsewhere, they're aligned with the same colors as the badge.

If the existing color doesn't match what's in the requirements, that just means we need to update the `-issue-[type]` color to be the appropriate one from the issue.